homeassistant/components/schlage/sensor.py

```diff
@@ -66,3 +56,35 @@ def _handle_coordinator_update(self) -> None:
         """Handle updated data from the coordinator."""
         self._attr_native_value = getattr(self._lock, self.entity_description.key)
         return super()._handle_coordinator_update()
+
+
+class SchlageBatterySensor(SchlageSensor):
+    """Schlage battery sensor entity."""
+
+
+class SchlageDurationSensor(SchlageSensor):
+    """Schlage duration sensor entity."""
+
+
+_SENSOR_DESCRIPTIONS: list[tuple[type[SchlageSensor], SensorEntityDescription]] = [
+    (
+        SchlageBatterySensor,
+        SensorEntityDescription(
+            key="battery_level",
+            device_class=SensorDeviceClass.BATTERY,
+            entity_category=EntityCategory.DIAGNOSTIC,
+            native_unit_of_measurement=PERCENTAGE,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+    ),
+    (
+        SchlageDurationSensor,
+        SensorEntityDescription(
+            key="auto_lock_time",
+            device_class=SensorDeviceClass.DURATION,
+            entity_category=EntityCategory.DIAGNOSTIC,
+            native_unit_of_measurement=UnitOfTime.SECONDS,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+    ),
+]
```

Comment:
```suggestion
_SENSOR_DESCRIPTIONS: list[SensorEntityDescription] = [
      SensorEntityDescription(
          key="battery_level",
          device_class=SensorDeviceClass.BATTERY,
          entity_category=EntityCategory.DIAGNOSTIC,
          native_unit_of_measurement=PERCENTAGE,
          state_class=SensorStateClass.MEASUREMENT,
      ),
      SensorEntityDescription(
          key="auto_lock_time",
          device_class=SensorDeviceClass.DURATION,
          entity_category=EntityCategory.DIAGNOSTIC,
          native_unit_of_measurement=UnitOfTime.SECONDS,
          state_class=SensorStateClass.MEASUREMENT,
      ),
]
```

Can you please move it back, where it was before

Reply:
I will revert this whole file. That block moved so I could reference the classes in the list of description objections